### ISSUE TITLE
Props not accurate in documentation

### ISSUE THREAD

**Issue by rektdeckard** [SorenHolstHansen/phosphor-leptos#1]
In order to get the example from the documentation to compile, I had to use `IconWeight` enum values directly, rather than static strings -- otherwise I get the following compiler error:

```
the trait bound `MaybeSignal<IconWeight>: From<&str>` is not satisfied
the following other types implement trait `From<T>`:
  <MaybeSignal<std::string::String> as From<&str>>
  <MaybeSignal<T> as From<ReadSignal<T>>>
  <MaybeSignal<T> as From<RwSignal<T>>>
  <MaybeSignal<T> as From<Memo<T>>>
  <MaybeSignal<T> as From<Signal<T>>>
  <MaybeSignal<T> as From<T>>
required for `&str` to implement `Into<MaybeSignal<IconWeight>>`
```

Additionally, I had to change the `size` value to `"32"` or `"32px"` to see expected rendering. I believe `"{32}"` is a mistake. The following was what worked for me:

```rs
#[component]
fn App() -> impl IntoView {
    view! {
        <Horse />
        <Heart color="#AE2983" weight={IconWeight::Fill} size="32px" />
        <Cube color="teal" size="32px" weight={IconWeight::Duotone} />
    }
}
```

**Context**

Rust nightly, with the following dependency versions:

```toml
[dependencies]
leptos = { version = "0.5.1", features = ["csr", "nightly"] }
phosphor-leptos = "0.1.0"
```

**Comment by SorenHolstHansen** [comment]
The IconWeight enum is on purpose, since I believe it is more idiomatic Rust, though perhaps I am wrong. 

Aaarh, but you mean the readme, of course. Yes I will change that. Just copied it from another README and forgot to update, thanks!

**Comment by SorenHolstHansen** [comment]
Fixed by #2
